# Single-cell Related Papers

This site contains candidate single-cell related papers which may be discussed later. Welcome to supplement.

## Review

* Stubbington, Michael JT, et al. "Single-cell transcriptomics to explore the immune system in health and disease." Science 358.6359 (2017): 58-63. [[link]](http://science.sciencemag.org/content/358/6359/58.long)

## Methods

* Kharchenko, Peter V., Lev Silberstein, and David T. Scadden. "Bayesian approach to single-cell differential expression analysis." Nature methods 11.7 (2014): 740-742. [[link]](https://www.nature.com/nmeth/journal/v11/n7/full/nmeth.2967.html)

*  Fan, Jean, et al. "Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis." Nature methods 13.3 (2016): 241-244. [[link]](https://www.nature.com/nmeth/journal/v13/n3/full/nmeth.3734.html)

* Qiu, Xiaojie, et al. "Reversed graph embedding resolves complex single-cell trajectories." Nature 201 (2017): 7.[[link]](https://www.nature.com/nmeth/journal/v14/n10/full/nmeth.4402.html)

* Rajaram, Satwik, et al. "Sampling strategies to capture single-cell heterogeneity." Nature methods 14.10 (2017): 967.[[link]](https://www.nature.com/nmeth/journal/v14/n10/full/nmeth.4427.html)

* Aibar, S., et al. (2017). "SCENIC: single-cell regulatory network inference and clustering." Nat Methods. [[link]](http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4463.html?WT.feed_name=subjects_biotechnology)

* Kiselev, Vladimir Yu, et al. "SC3: consensus clustering of single-cell RNA-seq data." Nature methods (2017). [[link]](https://www.nature.com/nmeth/journal/v14/n5/full/nmeth.4236.html)

* Qiu, Xiaojie, et al. "Single-cell mRNA quantification and differential analysis with Census." Nature methods 14.3 (2017): 309-315. [[link]](https://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4150.html)

* Bacher, Rhonda, et al. "SCnorm: robust normalization of single-cell RNA-seq data." Nature Methods 14.6 (2017): 584-586. [[link]](https://www.nature.com/nmeth/journal/v14/n6/abs/nmeth.4263.html)

* Roth, Andrew, et al. "Clonal genotype and population structure inference from single-cell tumor sequencing." Nature methods 13.7 (2016): 573-576. [[link]](https://www.nature.com/nmeth/journal/v13/n7/full/nmeth.3867.html)

* Shun H. Yip, Panwen Wang, Jean-Pierre A. Kocher, Pak Chung Sham, Junwen Wang; Linnorm: improved statistical analysis for single cell RNA-seq expression data, Nucleic Acids Research (2017) [[link]](https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx828#)

* Zhe Sun, Ting Wang, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu, Wei Chen; DIMM-SC: a Dirichlet mixture model for clustering droplet-based single cell transcriptomic data, Bioinformatics (2017) [[link]](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx490)

* Haghverdi, Laleh, et al. "Diffusion pseudotime robustly reconstructs lineage branching." Nature methods 13.10 (2016): 845-848. [[link]](https://www.nature.com/nmeth/journal/v13/n10/full/nmeth.3971.html)

## Technology and applications

* Svensson, Valentine, et al. "Power analysis of single-cell RNA-sequencing experiments." Nature methods (2017).[[link]](http://www.nature.com/nmeth/journal/v14/n4/full/nmeth.4220.html)

* Datlinger, Paul, et al. "Pooled CRISPR screening with single-cell transcriptome readout." Nature methods 14.3 (2017): 297-301. [[link]](http://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4177.html)

* Liu, Z., et al. (2017). "Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte." Nature advance online publication. [[link]](https://www.nature.com/nature/journal/vaop/ncurrent/full/nature24454.html)

* Enge, Martin, et al. "Single-Cell Analysis of Human Pancreas Reveals Transcriptional Signatures of Aging and Somatic Mutation Patterns." Cell (2017). [[link]](https://www.ncbi.nlm.nih.gov/pubmed/28965763?dopt=Abstract&utm_source=dlvr.it&utm_medium=twitter)

* van den Brink, S. C., et al. (2017). "Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations." Nat Meth 14(10): 935-936. [[link]](https://www.nature.com/nmeth/journal/v14/n10/full/nmeth.4437.html)

* Zheng, Chunhong, et al. "Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing." Cell 169.7 (2017): 1342-1356. [[link]](https://www.ncbi.nlm.nih.gov/pubmed/28622514?dopt=Abstract&utm_source=dlvr.it&utm_medium=twitter)

## Biorxiv

* Daniel A, Single cell transcriptional profiling reveals cellular diversity, communication, and sexual dimorphism in the mouse heart. Biorxiv [[link]](https://www.biorxiv.org/content/early/2017/10/12/201970?rss=1&utm_source=dlvr.it&utm_medium=twitter)

* Giovanni Iacono, et al. bigSCale: An Analytical Framework for Big-Scale Single-Cell Data. Biorxiv [[link]](https://www.biorxiv.org/content/early/2017/10/03/197244?rss=1&utm_source=dlvr.it&utm_medium=twitter)

* Jiarui Ding, et al. Interpretable dimensionality reduction of single cell transcriptome data with deep generative models. Biorxiv [[link]](https://www.biorxiv.org/content/early/2017/09/01/178624.full.pdf+html)

* Peter V. Kharchenko, et al. Accurate estimation of molecular counts in droplet-based single-cell RNA-seq experiments. Biorxiv [[link]](https://www.biorxiv.org/content/early/2017/09/13/171496)

* Donghyung Lee, et al. A robust statistical framework to detect multiple sources of hidden variation in single-cell transcriptomes. Biorxiv [[link]](https://www.biorxiv.org/content/early/2017/06/18/151217?rss=1&utm_source=dlvr.it&utm_medium=twitter)
