# Single-cell Related Papers

This site contains candidate single-cell related papers which may be discussed later. Welcome to supplement.


* Kharchenko, Peter V., Lev Silberstein, and David T. Scadden. "Bayesian approach to single-cell differential expression analysis." Nature methods 11.7 (2014): 740-742. [[link]](https://www.nature.com/nmeth/journal/v11/n7/full/nmeth.2967.html) I think this paper gives a very detailed description of scRNA-seq noise. But the method is hard to understand.

*  Fan, Jean, et al. "Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis." Nature methods 13.3 (2016): 241-244. [[link]](https://www.nature.com/nmeth/journal/v13/n3/full/nmeth.3734.html)

* Qiu, Xiaojie, et al. "Reversed graph embedding resolves complex single-cell trajectories." Nature 201 (2017): 7.[[link]](https://www.nature.com/nmeth/journal/v14/n10/full/nmeth.4402.html)

* Rajaram, Satwik, et al. "Sampling strategies to capture single-cell heterogeneity." Nature methods 14.10 (2017): 967.[[link]](https://www.nature.com/nmeth/journal/v14/n10/full/nmeth.4427.html)

* Aibar, S., et al. (2017). "SCENIC: single-cell regulatory network inference and clustering." Nat Methods. [[link]](http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4463.html?WT.feed_name=subjects_biotechnology)

* Kiselev, Vladimir Yu, et al. "SC3: consensus clustering of single-cell RNA-seq data." Nature methods (2017). [[link]](https://www.nature.com/nmeth/journal/v14/n5/full/nmeth.4236.html)

* Qiu, Xiaojie, et al. "Single-cell mRNA quantification and differential analysis with Census." Nature methods 14.3 (2017): 309-315. [[link]](https://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4150.html)

* Bacher, Rhonda, et al. "SCnorm: robust normalization of single-cell RNA-seq data." Nature Methods 14.6 (2017): 584-586. [[link]](https://www.nature.com/nmeth/journal/v14/n6/abs/nmeth.4263.html)

